src/OptionSettlement.sol

```diff
@@ -385,15 +347,13 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
             revert AlreadyClaimed(claimId);
         }
 
-        Option storage optionRecord = _option[optionId];
+        Option storage optionRecord = _option[_claimIdU160b];
```

Comment:
can we just make this _optionId and _claimId? the hungarian notation is kind of ugly here and doesn't really reflect anything useful to the reader.